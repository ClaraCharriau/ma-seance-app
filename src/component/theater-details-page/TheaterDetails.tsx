const TheaterDetails = () => {
    return <div>TheaterDetails</div>;
};

export default TheaterDetails;
