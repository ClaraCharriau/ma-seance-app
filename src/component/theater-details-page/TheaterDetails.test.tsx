import { render } from '@testing-library/react';
import TheaterDetails from './TheaterDetails';

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useLoaderData: () => {
        return {
            id: 1,
            name: 'UGC Normandie',
            address: '116 bis Av. des Champs-Élysées 75008 Paris',
            imgPath: '/ugc-normandie-paris'
        };
    }
}));

describe('Theater Details Component', () => {
    it('renders theater details component', () => {
        const component = render(<TheaterDetails />);

        expect(component.container).toMatchSnapshot();
    });
});
