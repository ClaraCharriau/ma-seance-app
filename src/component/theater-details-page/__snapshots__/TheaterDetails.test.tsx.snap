// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theater Details Component renders theater details component 1`] = `
<div>
  <div>
    TheaterDetails
  </div>
</div>
`;
